crates/tauri-cli/src/interface/rust.rs

```diff
@@ -1159,7 +1159,7 @@ pub(crate) fn get_cargo_target_dir(args: &[String]) -> crate::Result<PathBuf> {
     std::env::current_dir()?.join(target)
   } else {
     get_cargo_metadata()
-      .with_context(|| "failed to get cargo metadata")?
+      .with_context(|| "failed to fetch target directory from cargo metadata")?
```

Comment:
```suggestion
      .with_context(|| "failed to run `cargo metadata` command to get target directory")?
```
what about something like this? (same for the other one)

Reply:
Okay, sir. Thanks very much for your suggestion.